lighthouse-core/test/gather/fake-driver.js

```diff
@@ -5,9 +5,18 @@
  */
 'use strict';
 
-module.exports = {
-  getUserAgent() {
-    return Promise.resolve('Fake user agent');
+const browserVersion = {
+  protocolVersion: '1.3',
```

Comment:
this doesn't return `milestone`. Is this filename wrong and it's really not a fake driver but a fake connection?

Reply:
good catch. yeah the separation between protocol response vs driver method response got a little mixed up. 

Clarified this now.